packages/reporters/liquidation-reporter/liquidation-reporter.ts

```diff
@@ -99,43 +97,34 @@ export async function fetchUmaEcosystemData() {
         (info: any) => toChecksumAddress(info.id) === toChecksumAddress(financialContractAddress)
       )[0];
       // If there is no position info, then continue. This would happen if there is data that is not on the graph.
-      if (contractPositionsInfo == undefined) {
-        continue;
-      }
+      if (contractPositionsInfo == undefined) continue;
 
       // If the contract has no positions, we should also continue. Remove the contract from the list of active contracts
-      if (contractPositionsInfo.positions.length == 0) {
-        uniqueCollateralTypes[collateralAddress].activeFinancialContracts = removeItemFromArrayOfObjects(
```

Comment:
no longer need to use the `removeItemFromArrayOfObjects`. rather, we strip all empty elements later on.